test/parallel/test-tick-processor.js

```diff
@@ -59,6 +64,6 @@ function runTest(pattern, code) {
                         ' --prof-process --call-graph-size=10 ' + log,
                         {encoding: 'utf8'});
   assert(pattern.test(out));
-  fs.unlinkSync(log);
+  //fs.unlinkSync(log);
   fs.unlinkSync(testCodePath);
```

Comment:
I might have mis-commented before but this is backwards. The log file needs to be removed. The file in the tmp dir (`testCodePath`) does not. (The tmp directory is cleared by each test that needs it before it is used.)